.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ parameters:
     default: "v1.4.3"
   KUBECTL_VERSION:
     type: "string"
-    default: "v1.20.5"
+    default: "v1.19.9"
```

Comment:
a downgrade here?

Reply:
Yes, I noticed we are using 1.19 instead of 1.20 and IO wanted to make sure we are testing the proper version stack https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/values.yaml#L335

I'm keen to update it as well, so should I also propagate this update from 1.19 to 1.20 elsewhere?

Reply:
It doesn't really matter, I was just curious in case there was an error with v1.20.